crates/tessellation/src/geometry_builder.rs

```diff
@@ -524,19 +529,21 @@ impl GeometryBuilder for NoOutput {
 
 impl FillGeometryBuilder for NoOutput {
     fn add_fill_vertex(&mut self, _vertex: FillVertex) -> Result<VertexId, GeometryBuilderError> {
-        if self.count.vertices >= std::u32::MAX {
+        // This code is never ran!
```

Comment:
Do you mean that the case where count.vertices == std::u32::MAX is unreachable ?

Reply:
I misunderstood Clippy's suggestion, actually. It's rather the case where `count.vertices > std::u32::MAX` that's unreachable. I'll uncomment the code and change it to an equality.